Makefile

```diff
@@ -15,7 +15,7 @@ GOBIN=$(shell go env GOBIN)
 endif
 
 # Collect version information
-VERSION ?= $(shell git ls-remote --tags origin 'v*' | tail -1 | awk -F/ '{ print $$3 }')-next
+VERSION ?= $(shell git ls-remote --tags --refs origin 'v*' | awk -F/ '{ print $$3 }' | tail -1)-next
```

Comment:
This 68k Jeremy speaking, why not `tail` first and only `awk` the one line?

Reply:
I think it was just left over from switching out the change. I can switch back.